src/modules/ejector/ejector.py

```diff
@@ -98,7 +98,7 @@ def build_report(self, blockstamp: ReferenceBlockStamp) -> tuple:
         data, data_format = encode_data(validators)
 
         report_data = ReportData(
-            self.CONSENSUS_VERSION,
+            self.report_contract.get_consensus_version(blockstamp.block_hash),
```

Comment:
is it integer?

Reply:
Yes
`def get_consensus_version(self, block_identifier: BlockIdentifier = 'latest') -> int:`